test/e2e/serial/tests/workload_placement_tmpol.go

```diff
@@ -482,6 +482,8 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload placeme
 			updatedPod, err := wait.ForPodPhase(fxt.Client, pod.Namespace, pod.Name, corev1.PodRunning, 2*time.Minute)
 			if err != nil {
 				_ = objects.LogEventsForPod(fxt.K8sClient, pod.Namespace, pod.Name)
+				dumpNRTForNode(fxt.Client, targetNodeName, "target")
+				//TODO print scheduler events
```

Comment:
do you need events emitted by the scheduler or scheduler logs? The former, the most important among the former, should be emitted already by `LogEventsForPod`

Reply:
I actually meant scheduler logs, thanks for raising that, I'll address it in this PR

Reply:
scheduler logs are potentially very long. We dump events because they are concise. This change would probably need its own discussion and PR

Reply:
you're right, we'll need some filtering applied. I'll leave this change then.